src/components/icons/IndentIcon.tsx

```diff
@@ -0,0 +1,89 @@
+import { icon } from '../../../styled-system/recipes'
+import { token } from '../../../styled-system/tokens'
+import IconType from '../../@types/Icon'
+
+// eslint-disable-next-line jsdoc/require-jsdoc, react-refresh/only-export-components
```

Comment:
I'm not sure why `react-refresh/only-export-components` is needed here, since it is a component. does it need to be typed as `FC`?

Reply:
The eslint-disable comment was carried over from another file where it was needed, but it's redundant in this component and can be removed.